spec/API_specification/linear_algebra_functions.md

```diff
@@ -275,9 +275,35 @@ TODO
 TODO
 
 (function-svd)=
-### svd()
+### svd(x, /, *, compute_uv=True, full_matrices=True)
```

Comment:
Are we purposely deviating from NumPy's signature, in which `full_matrices` goes before `compute_uv`? Logically the choice here makes sense as `compute_uv` could be ignored, but I'd like to make sure this is a delibrate choice.

Reply:
These are keyword-only, and not positional, arguments, so the order in which they appear should not matter. Here, they are listed in alphabetical order.

Reply:
Ah yes, thanks @kgryte! My builtin parser automatically discarded / and * ...